src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  Component,
+  OnInit
+} from '@angular/core';
+
+import {
+  FormControl,
+  FormGroup
+} from '@angular/forms';
+
+@Component({
+  selector: 'app-phone-field-demo',
+  templateUrl: './phone-field-demo.component.html',
+  styleUrls: ['./phone-field-demo.component.scss']
+})
+export class PhoneFieldDemoComponent implements OnInit {
+
+  public phoneForm: FormGroup;
+
+  public phoneControl: FormControl;
```

Comment:
Check your alphabetical ordering of properties.

Reply:
Done